ansible/group_vars/all/creds.yml

```diff
@@ -10,3 +10,6 @@ sonic_default_passwords:
  - "YourPaSsWoRd"
  - "password"
 sonic_password: "password"
+
+k8s_master_login: "ubuntu"
+k8s_master_password: "admin"
```

Comment:
>admin [](start = 22, length = 5)

who create the default username and password? I thought default password is empty, is it true?

Reply:
default username is set by Ubuntu, password is set in Ansible job creating HA master

Reply:
Is it possible keep using default password (empty)? So we don't need a new password here.

---
In reply to: [513070790](https://github.com/Azure/sonic-mgmt/pull/2382#discussion_r513070790) [](ancestors = 513070790)

Reply:
I pushed change to let user choose own value for Ubuntu KVM for Kubernetes master; default username `ubuntu` is still used. Key vault can be used internally